.github/workflows/deploy.yml

```diff
@@ -17,8 +17,8 @@ jobs:
         with:
           repository: ${{ github.repository }}
           production_branch: ${{ github.event.repository.default_branch }}
-          build_artifact_name: "full-stack-app"
-          output_directory: "full-stack-app"
+          build_artifact_name: "static"
+          output_directory: "static"
```

Comment:
@EresDev will this break anything?

Reply:
Yes it will break. Probably both development & beta branch deployment. 

You have the old way to configure build and deploy scripts. But it only supports static.
And you have the new way which supports both full stack apps, and static only apps. 
But here in this config you are mixing both. As you have named it static to use old way, but old way do not have a parameter `statics_directory: "static"`

You can see examples of how to set these in the readme of https://github.com/ubiquity/cloudflare-deploy-action.

Maybe even bring ts-template to new style, and it will work for both static only and full stack apps. 
If you want to rename the term "full-stack-app", you can use any other word, but don't replace it with word "static" because it suits better with old static only way. 

Simple solution: abort this build & deploy script change.
(optional) Upgrade ts-template to use this new build & deploy config to support both static & full-stack-apps.

Reply:
Sorry but instructions unclear. I can revert this but I don't understand why

Reply:
Ok. I see that it failed because you didn't trigger the build before deploy. New deploy was looking for new "static" named artifact, but the artifact available was old one "full-stack-app". Triggering the build before deploy worked here. So, your update is not faulty, it is just a little confusing. 

why I don't want you to name it "static".

It is not reserved. You can call this static. Like here it is working now. 

But for this project, why are you calling it static, when it is not actually static, it contains backend functions too. 

This is useful when you want to understand how deploy action is building the directory structure to deploy the project. 

Using new full stack way:

```
full-stack-app:
- static
- functions
- other files & directories
```

Using old static only way:
```
static:
- index.html
- other static files and directories
```

Now what you are doing is:
```
static:
- static
- functions
- other files & directories
```

This way has a some potentional to be confusing when troubleshooting any problems at the build & deploy scripts. Which static directory it is and why it is causing problems when there is one.


We have 3 different parameters for 3 different purposes. There should be the difference in name. 

```
build_artifact_name: "static"
output_directory: "static"
statics_directory: "static"
```